lib/fs.js

```diff
@@ -1617,8 +1617,15 @@ function ReadStream(path, options) {
   if (!(this instanceof ReadStream))
     return new ReadStream(path, options);
 
+  if (options === undefined)
```

Comment:
It might be better to change the check on line 1624 to `else if (options === null || typeof options !== 'object')`